src/components/item/iterator.js

```diff
@@ -324,6 +369,42 @@ class ItemIterator extends Iterator {
   }
 }
 
+const DropTargetSpec = {
+  drop({ list, onItemImport }, monitor) {
+    const files = monitor
+      .getItem()
+      .files
+      .filter(isImageSupported)
+      .map(file => file.path)
+
+    onItemImport({ files, list })
+    return { files }
+  },
+
+  canDrop({ items }, monitor) {
+    switch (monitor.getItemType()) {
+      case DND.ITEMS:
+        return items.length > 1
+      case NativeTypes.FILE: {
+        return !!monitor.getItem().types
+          .find(type => isImageSupported({ type }))
+      }
+      default:
+        return false
+    }
+  }
+}
+
+const DropTargetCollect = (connect, monitor) => {
+  let isOver = monitor.isOver({ shallow: true }) && monitor.canDrop()
+  let type = monitor.getItemType()
+
+  return {
+    dt: connect.dropTarget(),
+    isOverFile: isOver && type === NativeTypes.FILE,
```

Comment:
`isOverFile` should also be true for dragging URLs

Reply:
done